sb_vision/find_3D_coords.py

```diff
@@ -0,0 +1,139 @@
+"""
+Handlers for detecting 3D marker Location.
+
+Also handles loading the calibration file which is passed into the
+location finding function.
+"""
+
+import functools
+import re
+from pathlib import Path
+from typing import List, Tuple, cast
+
+import cv2
+import numpy as np
+from lxml import etree
+
+from sb_vision.coordinates import Cartesian, PixelCoordinate
+
+
+def _get_values_from_xml_element(element: etree.Element) -> List[str]:
+    """Parse an xml tag with space-separated variables."""
+    text = []  # type: List[str]
+    for e in element.itertext():
+        e = e.strip()
+        if e:
+            text += re.split(r'\s+', e)
+    return text
+
+
+def convert(element: etree.Element) -> List[List[np.float64]]:
```

Comment:
Is there really not a better name for this? (Also: should it be private, i.e: `_`-prefixed?)

Reply:
fixed in fcbd21e77acd6c677d8b24fde4921beffe29d871